doc/Usage.adoc

```diff
@@ -0,0 +1,272 @@
+= Usage
+
+:toc:
+:toc-placement!:
+
+The following sections explain how you can use the Landrush DNS server and customize it as per your requirements, once you have link:README.adoc[installed] it.
+
+'''
+toc::[]
```

Comment:
Odd, I still don't see the table of content when I view this whole file. The same with toubleshooting. When I view the other documents I see the toc. Any idea? One difference I can see is that in this document where it does not work, there is an extra blank lines after the top heading, in the documents where it works, the :toc: comes directly after the heading (no blank line).